templates/main/index.html.ep

```diff
@@ -19,7 +19,6 @@
     </div>
 </div>
 
-<a name="build-results"/>
```

Comment:
We need an anchor here for our QA dashboards:

- https://openqa.opensuse.org/?group=openSUSE%20Tumbleweed$|openSUSE%20Leap%20[0-9]{2}.[0-9]$|openSUSE%20Leap.*JeOS$|openSUSE%20Krypton|openQA&limit_builds=1&time_limit_days=14#build-results